### PR TITLE
🌐 Add Korean translation for Tutorial - Extra Models

### PR DESCRIPTION
**Pull request by peebbv6364**
Hello
I translated the Extra Models part of the tutorial.
Please review if there is anything wrong. thanks.

This PR translates `tutorial/extra-models.md` in Korean.
related: #2017

### CODE REVIEW

File: docs/ko/docs/tutorial/extra-models.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+!!! 참고
+    <a href="https://pydantic-docs.helpmanual.io/usage/types/#unions" class="external-link" target="_blank">`Union`</a>을 선언할때, 가장 특정한 타입을 첫번째로 포함하고, 덜 특정한 타입을 뒤에 붙이세요. 예를 들어, `Union[PlaneItem, CarItem]`에서 더 특정한 `PlaneItem`은 `CarItem`보다 더 앞에 옵니다.
+
+```Python hl_lines="1  14-15  18-20  33"
+{!../../../docs_src/extra_models/tutorial003.py!}
+```
+
+## 모델들의 리스트
+
+같은 방법으로, 당신은 객체의 리스트를 응답으로 선언할 수 있습니다.
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/extra-models.md:174]:
```suggestion
같은 방법으로, 객체의 리스트를 응답으로 선언할 수 있습니다.
```

File: docs/ko/docs/tutorial/extra-models.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+{!../../../docs_src/extra_models/tutorial004.py!}
+```
+
+## 임의의 `dict`로 응답하기
+
+당신은 Pydantic 모델을 사용하지 않고 단순히 키와 값의 타입으로 선언하는 평범한 임의의 `dict`로 응답을 선언할 수 있습니다.
+
+당신이 미리 유효한 필드와 속성의 이름(이것은 아마 Pydantic 모델에 필요할 것입니다)을 알지 못할때 유용합니다.
+
+이 상황에서 당신은 `typing.Dict`를 사용할 수 있습니다.:
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/extra-models.md:188]:
```suggestion
Pydantic 모델을 사용하지 않고 단순히 키와 값의 타입으로 선언하는 평범한 임의의 `dict`로 응답을 선언할 수 있습니다.

미리 유효한 필드와 속성의 이름(이것은 아마 Pydantic 모델에 필요할 것입니다)을 알지 못할때 유용합니다.

이 상황에서 `typing.Dict`를 사용할 수 있습니다.:
```

File: docs/ko/docs/tutorial/extra-models.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+```Python hl_lines="1  8"
+{!../../../docs_src/extra_models/tutorial005.py!}
+```
+
+## 정리
+
+각각의 상황에서 다수의 Pydantic 모델들을 사용하고 자유롭게 상속합니다.
+
+만약 개체가 다른 "상태(states)"를 가질 수 밖에 없다면, 굳이 하나의 개체당 하나의 데이터 모델을 가질 필요가 없습니다. `password`, `password_hash`는 있으나 비밀번호는 없는 사용자 "entity" 상황이 그렇습니다.
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/extra-models.md:198]:
```suggestion
각각의 상황에서 다수의 Pydantic 모델들을 사용하고 자유롭게 상속하십시오.

만약 개체가 다른 "상태"를 가질 수밖에 없다면, 굳이 하나의 개체당 하나의 데이터 모델을 가질 필요가 없습니다. `password`, `password_hash`는 있으나 비밀번호는 없는 사용자 "entity" 상황이 그렇습니다.
```

**Review by hard-coders — COMMENTED** [review]
Welcome aboard 🎉

It's a great job, and I really appreciate your work!
Please see the reviews and tell me what you think.

File: docs/ko/docs/tutorial/extra-models.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+중복 코드 제거는 **FASTAPI**의 핵심 개념 중 하나입니다.
+
+코드 중복은 버그의 가능성, 보안 이슈, 코드 비동기화 문제(한 장소에는 업데이트 하지만 다른 곳에서는 그렇지 못할때) 등을 증가시킵니다.
+
+그리고 이러한 모델들은 많은 데이터를 공유하고 타입과 인자 이름들을 반복적으로 사용하게 됩니다.
+
+우리는 더 잘할 수 있습니다.
+
+우리는 다른 모델들의 기초가 되는 `UserBase` 모델을 선언할 수 있습니다. 그리고 우리는 이 인자들(타입 선언, 검증 등)을 상속받은 모델인 서브클래스를 만들 수 있습니다.
```
[the remark this answers is not in the document]
Comment (peebbv6364) [reply docs/ko/docs/tutorial/extra-models.md]: I think I'd better get rid of the expression "우리". thanks

**Comment by peebbv6364** [comment]
@hard-coders 
Thank you for your review.
I accepted your points and pushed the new commit.

File: docs/ko/docs/tutorial/extra-models.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
+# 추가 모델
+
+이전의 예제에 이어서, 일반적으로는 한 가지 이상의 관계 모델을 가질 것입니다.
+
+사용자모델의 경우가 특히 그렇습니다. 왜냐하면:
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/extra-models.md:5]:
```suggestion
이전의 예제에 이어서, 일반적으로는 한 가지 이상의 관계 모델을 가질 것입니다.

사용자 모델의 경우가 특히 그렇습니다. 왜냐하면:
```

**Review by hard-coders — COMMENTED** [review]
One thing is left to do.

You must modify `docs/ko/mkdocs.yml`

**Review by hard-coders — APPROVED** [review]
It seems perfect. 🚀

**Pull request closed without merging**